frontend/src/scenes/experimentation/FeatureFlags.js

```diff
@@ -53,20 +58,40 @@ function _FeatureFlags() {
         },
         {
             title: 'Usage',
-            render: function RenderActive(featureFlag) {
+            render: function RenderUsage(featureFlag) {
                 return (
-                    <LinkButton
+                    <Link
                         to={
                             '/insights?events=[{"id":"$feature_flag_called","name":"$feature_flag_called","type":"events"}]&properties=[{"key":"$feature_flag","value":"' +
```

Comment:
Unrelated to this PR, but I'm thinking it might be worth doing this on active users instead of volume?

Reply:
Done